### PR TITLE
Update Junit tests for Activity Create Command

### PR DESCRIPTION
**Pull request by ivanaitzliddat**
Remove redundant tests
Add new tests
Edit the name of the constants

### CODE REVIEW

**Review by froststein — APPROVED** [review]
Looks good to merge

File: src/test/java/seedu/splitlah/command/ActivityCreateCommandTest.java [hunk 1]
```diff
@@ -137,4 +117,62 @@ public void prepare_hasMissingArguments_InvalidCommand() {
         assertEquals(InvalidCommand.class, activityWithMissingInvolvedListArgument.getClass());
     }
 
+
```
Comment (warrencxw) [inline src/test/java/seedu/splitlah/command/ActivityCreateCommandTest.java:120]: Remove newline for standardisation

File: src/test/java/seedu/splitlah/command/ActivityCreateCommandTest.java [hunk 2]
```diff
@@ -137,4 +117,62 @@ public void prepare_hasMissingArguments_InvalidCommand() {
...
+    /**
+     * Checks if an activity is created when an activity has duplicate names in the involved list.
+     * @throws InvalidDataException If there are no sessions stored or
```
Comment (warrencxw) [inline src/test/java/seedu/splitlah/command/ActivityCreateCommandTest.java:123]: Consider adding a newline between description and the `@throws` annotation.

Comment (warrencxw) [reply src/test/java/seedu/splitlah/command/ActivityCreateCommandTest.java:123]: Consider rewording the description as the current description implies that you are expecting an activity to be created, contrary to the test.

File: src/test/java/seedu/splitlah/command/ActivityCreateCommandTest.java [hunk 3]
```diff
@@ -137,4 +117,62 @@ public void prepare_hasMissingArguments_InvalidCommand() {
...
+    public void run_hasNameDuplicatesInInvolvedList_activityListSizeRemainsOne() throws InvalidDataException {
+        String userInput = "activity /create /sid 1 /n Dinner /p Alice /i Alice Alice Charlie /co 30";
+        Command command = Parser.getCommand(userInput);
+        assertEquals(ActivityCreateCommand.class, command.getClass());
+        command.run(manager);
+        assertEquals(1, manager.getProfile().getSession(1).getActivityList().size());
+    }
+
+    /**
+     * Checks if activity unique identifier is incremented if an activity fails
```
Comment (warrencxw) [inline src/test/java/seedu/splitlah/command/ActivityCreateCommandTest.java:136]: Consider rewording the description as current description implies that you are expecting an increment to the identifier, contrary to the test.

**Review by warrencxw — CHANGES_REQUESTED** [review]
There are several phrasing issues which could be changed to improve the clarity of the test methods. Also, would be good if a new line is added between each JavaDoc comment description and the annotations.

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Pull request merged**